#[117835155] Taskee skillset

akabiru opened the pull request:
#### What does this PR do?

Enable `taskee` to update own skillsets
#### How should this be manually tested?
- Log in as taskee
- click on `My Skillset` in dashboard
- You can now add or delete and existing skillset
#### What are the relevant pivotal tracker stories?
#117835155
#### Screenshots

![](https://www.dropbox.com/s/ujs3yj6v4nck46q/Screen%20Shot%202016-04-26%20at%204.17.47%20PM.png?raw=1)

![](https://www.dropbox.com/s/k9od70rmy7673vp/Screen%20Shot%202016-04-26%20at%204.18.19%20PM.png?raw=1)

![](https://www.dropbox.com/s/rj97sggwpd3rldy/Screen%20Shot%202016-05-03%20at%2011.55.19%20AM.png?raw=1)

Commit:
feat(taskee skillset): add taskee controller

Commit:
feat(taskee skillset): add taskee skillset

 - taskee is able to add skillsets

[Starts #117835155]

Commit:
Merge branch 'master' of https://github.com/andela/workdey into ft-taskee-skillset-#117835155

Commit:
feat(taskee skillset): user taskee add skillset

 - only taskee can update own profile
 - add utility method `taskee?`and `tasker?` to check
   user type
 - fix rubocop issues

[Finishes #117835155]

codesword commented on app/assets/stylesheets/styles/_dashboard.scss:
```diff
@@ -92,118 +92,117 @@ nav .nav-wrapper {
-        margin-top: em(7.5px);
-        border-radius: 100%;
-      }
+  .dropdown {
```
Good job cleaning up this css file.

Commit:
feat(taskee skillset): refactor controller

 - move controller logic from task to skillset
 - add toast notification on duplicate entry

[Finishes #117835155]

Commit:
fix(toast notification): incorrect notifications

 - get skillsets from curent user
 - fix find of skillset

[Finishes #118836815]

Commit:
Merge branch 'master' into ft-taskee-skillset-#117835155

Commit:
chore(taskee skillset): write tests for feature

 - move `login_with` to Helpers module
 - write feature tests for taskee skillset
 - write controller specs for #index

[#117835155]

Commit:
test(taskee skillsets):routing and ctrl tests

[#117835155]

Commit:
chore(taskee skillset): fix hound ci issues

 - remove unused variable
 - fix style violations

[#117835155]

Commit:
Merge branch 'master' of https://github.com/andela/workdey into ft-taskee-skillset-#117835155

lekansogunle commented on spec/controllers/skillsets_controller_spec.rb:
```diff
@@ -0,0 +1,59 @@
+
+    context "when the task does not exist" do
+      it "should create a task" do
+        task = create(:task, name: "Cleaning")
```
Nice!

lekansogunle replied:
Try using a generic name for this with https://github.com/stympy/faker#fakername.

akabiru replied:
@andela-osogunle For the sake of clarity in the test. I chose to define it because `Faker` will generate a person's name instead of a `task`?

lekansogunle replied:
Then use a random word. Faker::Lorem.words

lekansogunle commented on spec/helpers/skillsets_helper_spec.rb:
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
```
Can we remove these empty test files since they are not being used presently.

akabiru replied:
@andela-osogunle Sure thing.

lekansogunle commented on spec/helpers/tasks_helper_spec.rb:
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
```
Can we remove these empty test files since they are not being used presently.

Commit:
chore refactor skillset ctrl spec

 - use faker instead of literal string
 - remove unused spec helper files

[Finishes #117835155]
